### PR TITLE
ADGetUser - enable script execution with "Active Directory Query" instances only

### PR DESCRIPTION
**Pull request by noaco**
fixes https://github.com/demisto/etc/issues/16366

### CODE REVIEW

File: Scripts/script-ADGetUser.yml [hunk 1]
```diff
@@ -189,7 +189,12 @@ script: |-
   else:
       resp = listAll(attrs, demisto.get(demisto.args(),'using'), userAccountControlOut)
 
+  expectedErrs = ['invalid attribute type dn', 'Unsupported Command']
```
Comment (anara123) [inline Scripts/script-ADGetUser.yml]: expectedErrs => expected_errs
errData => err_data

File: TestPlaybooks/playbook-ADGetUser_fails_with_v2-Test.yml [hunk 2]
```diff
@@ -0,0 +1,187 @@
+id: Test ADGetUser Fails with no instances 'Active Directory Query' (old version)
+version: 18
```
Comment (anara123) [inline TestPlaybooks/playbook-ADGetUser_fails_with_v2-Test.yml]: 18 => -1

File: Scripts/script-ADGetUser.yml (hunk 1 above)
Comment (noaco) [reply Scripts/script-ADGetUser.yml]: used the same style as appears on the script...

**Comment by anara123** [comment]
```
Test ADGetUser Fails with no instances 'Active Directory Query' (old version) failed with error/s
Playbook Test ADGetUser Fails with no instances 'Active Directory Query' (old version) has failed:
	- Command: !ADGetUser extend-context="errData="
	- Body: Error from Scripts is : Command ADGetUser was not found in module supports list (88)
	- Command: Executing conditions:
Label: yes, Condition: [null Equals No instances of "Active Directory Query" are configured.
If you are trying to run "ADGetUser" with "Active Directory Query V2", please use the aquivilant command "ad-get-user" instead.]
	- Body: The output [] didn't satisfy any of the conditions of task 3 (31)
```

**Pull request merged**